time time_stamp  # Message 발행 시간 (ros::Time)
geometry_msgs/msg/Vector3 accel
geometry_msgs/msg/Vector3 gyro
geometry_msgs/msg/Vector3 mag 
smart_cargo_package/imu_angle_mssgs angle